cosmotheka/mappers/mapper_Quaia.py

```diff
@@ -0,0 +1,152 @@
+from .utils import get_map_from_points
+from .mapper_base import MapperBase
+from astropy.table import Table
+import numpy as np
+import healpy as hp
+
+
+class MapperQuaia(MapperBase):
+    """
+    """
+    map_name = 'Quaia'
+
+    def __init__(self, config):
+        self._get_defaults(config)
+        self.rot = self._get_rotator('C')
+        self.num_z_bins = config.get('num_z_bins', 500)
+        self.z_edges = config.get('z_edges', [0, 4.5])
+        self.zbin_name = 'z%.3lf_%.3lf' % (self.z_edges[0], self.z_edges[1])
+        self.z_name = config.get("z_name", "redshift_quaia")
+
+        self.cat_data = None
+        self.npix = hp.nside2npix(self.nside)
+
+        self.nl_coupled = None
+        self.mskflag = None
+        self.ipix = None
+        self.dndz = None
+
+    def _get_catalog(self):
+        # Returns the mapper's catalog \
+        # after binning it in redshift.
+
+        cat = Table.read(self.config['data_catalog'], format='fits')
+        cat.keep_columns(['ra', 'dec', self.z_name, self.z_name+'_err'])
+        z_d = cat[self.z_name]
+        mask_bin = (z_d < self.z_edges[1]) & (z_d >= self.z_edges[0])
+        cat = cat[mask_bin]
+        return cat.as_array()
+
+    def get_catalog(self):
```

Comment:
Annyoing that we have to define this per mapper...

Reply:
every dataset has its quirks